### PR TITLE
Added functional test for blazor admin unauthorized page for AB#13668

### PR DESCRIPTION
**Pull request by rayking29**
# Implements [AB#13668](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/13668)

## Description

Added functional test for blazor admin unauthorized page.

## Testing

- [ ] Unit Tests Updated
- [x] Functional Tests Updated
- [ ] Not Required

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

File: Apps/Admin/Tests/Functional/cypress.config.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ module.exports = defineConfig({
             idir_password: "",
             keycloak_username: "blazoradmin",
             keycloak_password: "",
+            unauthorized_keycloak_username: "healthgateway",
```
Comment (BrianMaki) [inline Apps/Admin/Tests/Functional/cypress.config.js:22]: Can we change this to 'keycloak_unauthorized_username'?  This seems to be the standard for Healthgateway functional tests.

Screen Shot 2022-08-15 at 3 29 33 PM

**Review by BrianMaki — CHANGES_REQUESTED** [review]
Small change.

File: Apps/Admin/Tests/Functional/cypress/integration/e2e/unauthorized.cy.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { skipOn } from "@cypress/skip-test";
+
+const username = Cypress.env("unauthorized_keycloak_username");
+const password = Cypress.env("keycloak_password");
+
+describe("Unauthorized", () => {
+    it("Verify unauthorized user reaches page.", () => {
+        skipOn("localhost");
```
Comment (MikeLyttle) [inline Apps/Admin/Tests/Functional/cypress/integration/e2e/unauthorized.cy.js:8]: This test should work on localhost if it's not logging in with IDIR.

Comment (rayking29) [reply Apps/Admin/Tests/Functional/cypress/integration/e2e/unauthorized.cy.js:8]: I tested this and it still doesn't work.  It gets a cross-origin error the same as the IDIR login.

**Review by MikeLyttle — APPROVED** [review]

**Review by BrianMaki — APPROVED** [review]

**Pull request merged**